Content.Server/Chemistry/ReagentEffects/MakeSentient.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+//using Content.Server.AI.Components; // This was in the original code but caused an error so I'm gonna keep it as a comment just in case
+using Content.Server.Mind.Components;
+using Content.Server.NPC.Components; // Don't think I actually need this but keeping it just in case
```

Comment:
What IDE are you using? VS or Rider should just handle changing these for you.

Reply:
Not quite sure what you mean, but my IDE is Visual Studio Code for the actual code editing. I also use Visual Studio 2022 for actually building and launching the game. I don't know how to get the used components to change automatically using either IDE, if that's what you're talking about.